src/api/v1/model_monitors.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Union
+from src.middleware.auth import authenticate_user
+from src.schemas.modelMonitor import ModelMonitor, ModelMonitorCreateDto
+from fastapi import APIRouter, Depends, status
+from src import crud
+from sqlalchemy.orm import Session
+from src.core.db import get_db
+from src.schemas.user import User
+from src.utils.errors import add_error_responses, errors
+
+
+model_monitors_router = APIRouter()
+
+
+@model_monitors_router.post(
+    "/model-monitors",
+    tags=["Model Monitors"],
+    response_model=ModelMonitor,
+    summary="Create a model monitor",
+    status_code=status.HTTP_201_CREATED,
+    responses=add_error_responses([400, 401, 409]),
+)
+async def create_model_monitor(
+    body: ModelMonitorCreateDto,
+    db: Session = Depends(get_db),
+    authenticated_user: User = Depends(authenticate_user),
+) -> ModelMonitor:
+
+    if body is not None:
+        new_model_monitor = crud.model_monitors.create(db=db, obj_in=body)
+        return new_model_monitor
+    else:
+        return errors.bad_request("Body should not be empty")
```

Comment:
I think the check here is not needed. Pydantic will catch this right?

Reply:
Fixed!